### ISSUE TITLE
Screen redraw problem

### ISSUE THREAD

**Issue by rwhitworth** [jesseduffield/lazydocker#52]
**Describe the bug**
A partial screen redraw problem exists where the Config pane is overwritten with a previous confirmation box

**To Reproduce**
1. On OSX (not tested elsewhere) load lazydocker
2. Click on Images pane
3. Hit 'D' to prune unused images
4. Hit '?' (because I didn't know what key to press)
5. Notice the confirmation box text is now gone, but outline of confirmation box remains
6. Hit 'enter' or 'escape' or 'q'
7. See 'you want to prune all unused images?' confirmation box covering part of Config pane

**Expected behavior**
On step 5 or 6 I expect the confirmation box to disappear

**Screenshots**
![image](https://user-images.githubusercontent.com/7214365/60451814-f5d4bd00-9bfa-11e9-83e0-5e5d05a8db85.png)

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Lazydocker Version 0.2.5 (via homebrew)
 - The last commit id if you built project from sources (run : ```git-rev parse HEAD```)

**Additional context**
n/a

**Comment by PikachuEXE** [comment]
Install `0.3` and try again?

**Comment by mjarkk** [comment]
@PikachuEXE This is not yet fixed in `0.3`  
I'll take a look at this and see if i can fix this.

**Comment by jesseduffield** [comment]
Seems the underlying issue is that prevView is not set on confirmation panels because we don't want to e.g. focus the main panel, then bring up a confirmation panel, then bring up a menu, and then return to the confirmation panel, but forget that we came from the main panel in the first place. Perhaps we need to have a stack for the previously focused views, but that does sound very complicated. I'm tempted to just restrict pressing '?' on confirmation panels for now and instead show the keybindings at the bottom left.

I've got a PR up for this here: https://github.com/jesseduffield/lazydocker/pull/124
